Fixed #36329 -- Removed non-code custom link text when cross-referencing Python objects. (#19386)
django/django: docs/intro/tutorial07.txt

it easier for your database to do the search.

Now's also a good time to note that change lists give you free pagination. The

> default is to display 100 items per page. Features like
> :attr:`~django.contrib.admin.ModelAdmin.list_per_page` for pagination,
> :attr:`~django.contrib.admin.ModelAdmin.search_fields` for search boxes,
> :attr:`~django.contrib.admin.ModelAdmin.list_filter` for filters,
> :attr:`~django.contrib.admin.ModelAdmin.date_hierarchy` for date hierarchies,
> and :attr:`~django.contrib.admin.ModelAdmin.list_display` for column header
> ordering all work together as expected.

Reviewer: ```suggestion
default is to display 100 items per page. This can be customized via the
:attr:`~django.contrib.admin.ModelAdmin.list_per_page` attribute.
```
I am very tempted to simplify this. I don't know why `date_hierarchy` is included considering it's not been introduced in the tutorial and this sentence seems a little pointless (this works together like you'd expect, so why do we need to point this out :thinking:)

Author: Agreed! The reader has no reason to expect that the features should'nt work together, since they'd have used them *together* in this tutorial. And `date_hierarchy` can be excluded.

make it easier for your database to do the search.

> Now's also a good time to note that change lists give you free pagination. The

Reviewer 2: Did you consider keeping these links somehow? E.g. "search boxes (:attr:`<django.contrib.admin.ModelAdmin.search_fields>`)"

Reviewer 3: I added a `seealso`. Sorry, I had forgotten to change this one
